spec/entities/base_spec.rb

```diff
@@ -51,37 +59,103 @@
   end
 
   describe 'request' do
-    let(:fixture) { File.read("spec/savon/fixtures/project/does_not_exists.xml") }
-    let(:client) { mock('client') }
-    let(:response) { mock('response') }
-    let(:request) { mock('request') }
-    let(:options) { mock('options') }
+    context 'with successful responses' do
+      let(:exists_response) { {'exists' => false} }
+      let(:bases_response) { {'bases' => {'id' => 1}} }
+      let(:prefix_bases_response) { {'bases' => {'id' => 2}} }
+
+      let(:connection) { Faraday.new { |builder| builder.adapter :test do |stubs|
```

Comment:
If someday this stub starts to get unused it will never fail and will remain there forever. Since our specs are our documentation, verifying this is stub is our way of forcing it to stay up to date.